src/include/smf/rpc_client.h

```diff
@@ -6,10 +6,11 @@
 #include <utility>
 #include <vector>
 #include <optional>
+#include <map>
```

Comment:
Is this header needed?

Reply:
I will check.

Reply:
Ok, I had included it because it seems to me that this directive is missing inside of the seastar tls header files (***seastar/net/tls.hh***) causing:
```bash 
In file included from ../../src/include/smf/rpc_client.h:12,
                 from ../../src/core/rpc_client.cc:3:
smf-deps-install/include/seastar/net/tls.hh:297:14: error: ‘multimap’ in namespace ‘std’ does not name a template type
  297 |         std::multimap<sstring, boost::any> _blobs;
      |              ^~~~~~~~
smf-deps-install/include/seastar/net/tls.hh:34:1: note: ‘std::multimap’ is defined in header ‘<map>’; did you forget to ‘#include <map>’?
   33 | #include <seastar/net/api.hh>
  +++ |+#include <map>
   34 | #include "../core/internal/api-level.hh"
```  
This should be a seastar bug, which doesn`t seem to show up in redpanda due to the order in which this headers are included. Even if ```seastar/net/tls.hh``` is included after  ```seastar/net/tls.hh``` the problem seems to vanish. Thus we have three choices:
1- change include order (that could kind of hide the problem)
2- include <map> header maybe with a note
3- fix the problem inside seastar itself (problem is that i am using the latest stable release of the library, not the current master)
Let me know wdyt.

Reply:
makes sense